l10n_ch_zip/__openerp__.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Nicolas Bessi, Olivier Jossen, Guewen Baconnier
+#    Copyright Camptocamp SA
+#    Copyright brain-tec AG
```

Comment:
Seems to be the case on the whole repository. I added the dates in these modules but one should do another PR to correct the other modules.